tests/fixed/annotation-name.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+interface Foo
+{
+    /**
+     * This method does something
+     */
+    public function doSomething(): void;
+
+    /**
+     * This method does not perform any action
+     */
+    public function nothing(): void;
+}
+
+final class Bar implements Foo
+{
+    /** @inheritDoc */
```

Comment:
I think this would get removed because it does not help with for instance, understanding an iterable argument. Trying adding a list of strings as argument to `doSomething`?